components/blocks/inlineCallout.js

```diff
@@ -16,8 +16,16 @@ const InlineCallout = ({ children, icon, color, bold, href }) => {
       : color === "l-blue-70"
       ? styles.CloudText
       : styles.KBText;
+  const bordercolor =
```

Comment:
Instead of such big ternary operations, we should use switch case.

Reply:
I have made the necessary changes